content/docs/user-guide/data-management/track-sync-version.md

```diff
@@ -0,0 +1,178 @@
+# Track, Sync, and Version Data
```

Comment:
Good stuff, @jorgeorpinel . But let's please step back and discuss the bigger picture first. How is it (expect being more low level) from all the get started, tutorials, etc? What would be the purpose of this page

Reply:
Sounds good. I just wanted to get moving with all of this and ended up writing very final-looking docs (around one PR per day), but I realize we may need to reconsider a lot of the content, the approach, conceptual level, etc. In that sense these are still draft-ish. And thanks

Reply:
> How is it (different) from all the get started, tutorials, etc? What would be the purpose of this page?

This page is more comprehensive than the GS. It's not a tutorial because we'll keep this up to date regularly, and doesn't use a real-life example

- GS: high-level; strategic features only; Purpose: onboard
- UG: mixed; comprehensive (feature set scope); **Purpose: understand**
- CR: low-level; comprehensive (command scope); Purpose: usage

> Refer to our current [docs framework](https://www.notion.so/iterative/Framework-7e86e484e50e42529df3dbfe32b4e987) proposal.

Reply:
> p.s. It may be more of a priority to review https://github.com/iterative/dvc.org/pull/4058 though, @shcheklein (after https://github.com/iterative/dvc.org/issues/2856).